Loading checkpoint shards:   0%|          | 0/2 [00:00<?, ?it/s]Loading checkpoint shards:  50%|█████     | 1/2 [00:49<00:49, 49.93s/it]Loading checkpoint shards: 100%|██████████| 2/2 [01:05<00:00, 29.63s/it]Loading checkpoint shards: 100%|██████████| 2/2 [01:05<00:00, 32.68s/it]
/cbica/home/xjia/.conda/envs/textlearning/lib/python3.9/site-packages/transformers/generation/configuration_utils.py:381: UserWarning: `do_sample` is set to `False`. However, `temperature` is set to `0.9` -- this flag is only used in sample-based generation modes. You should set `do_sample=True` or unset `temperature`. This was detected when initializing the generation config instance, which means the corresponding file may hold incorrect parameterization and should be fixed.
  warnings.warn(
/cbica/home/xjia/.conda/envs/textlearning/lib/python3.9/site-packages/transformers/generation/configuration_utils.py:386: UserWarning: `do_sample` is set to `False`. However, `top_p` is set to `0.6` -- this flag is only used in sample-based generation modes. You should set `do_sample=True` or unset `top_p`. This was detected when initializing the generation config instance, which means the corresponding file may hold incorrect parameterization and should be fixed.
  warnings.warn(
Loading checkpoint shards:   0%|          | 0/2 [00:00<?, ?it/s]Loading checkpoint shards:  50%|█████     | 1/2 [00:29<00:29, 29.23s/it]Loading checkpoint shards: 100%|██████████| 2/2 [00:39<00:00, 18.11s/it]Loading checkpoint shards: 100%|██████████| 2/2 [00:39<00:00, 19.78s/it]
Traceback (most recent call last):
  File "/gpfs/fs001/cbica/home/xjia/qlora/examples/vicuna_generate.py", line 105, in <module>
    test_df = pd.read_csv('/cbica/home/xjia/qlora/data/lab/test_mini.csv')
NameError: name 'pd' is not defined
